### PR TITLE
Module "tournament2": Fix several PHP Warnings and one PHP Fatal error

### PR DESCRIPTION
**Pull request by andygrunwald**
## Context

This PR is fixing several PHP Warnings in the `tournament2` module.

## Dependencies

This one depends on https://github.com/lansuite/lansuite/pull/653
https://github.com/lansuite/lansuite/pull/653 need to be merged before.

### CODE REVIEW

File: modules/tournament2/breaks.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 $dsp->NewContent(t('Turnier - Pausenverwaltung'), t('Trage Startzeitpunkt und dauer der Pause ein. Lansuite wird entsprechend spätere Rundenzeiten berechnen'));
 
+$stepParameter = $_GET['step'] ?? 0;
 switch ($_GET['step']) {
```
Comment (M4LuZ) [inline modules/tournament2/breaks.php:6]: I think the intent to replace the next line with $stepParameter?

**Review by M4LuZ — APPROVED** [review]

File: modules/tournament2/breaks.php (hunk 1 above)
Comment (andygrunwald) [inline modules/tournament2/breaks.php:6]:
```suggestion
switch ($stepParameter) {
```

Comment (andygrunwald) [reply modules/tournament2/breaks.php]: You are correct. Fixed in https://github.com/lansuite/lansuite/pull/657/commits/71930e5cc5033824b707ff6c7b09aa3c86df0d93

**Comment by andygrunwald** [comment]
Merging through, because the approval was given.

**Pull request merged**